controller/search_spaces_blackbox_test.go

```diff
@@ -183,9 +245,23 @@ func differentValues() expect {
 	}
 }
 
+func sortedSpaces(expectedSpaces ...string) expect {
+	return func(t *testing.T, scenario okScenario, result *app.SearchSpaceList) {
+		for i, space := range expectedSpaces {
+			if *result.Data[i].Attributes.Name != space {
+				t.Errorf("got %s, wanted %s", *result.Data[i].Attributes.Name, space)
+			}
+
+		}
+	}
+}
+
 func limit(n int) *int {
 	return &n
 }
 func offset(n string) *string {
 	return &n
 }
+func sortPtr(n string) *string {
```

Comment:
This is not needed. We have the `ptr` package that provides a `String` function that does exactly what `sortPtr` does. Also `offset` and `limit` should be replaced by `ptr.String` and `ptr.Int` as well.

Reply:
done